app/selenium_ui/jira/pages/pages.py

```diff
@@ -6,6 +6,10 @@
 from selenium_ui.jira.pages.selectors import UrlManager, LoginPageLocators, DashboardLocators, PopupLocators, \
     IssueLocators, ProjectLocators, SearchLocators, BoardsListLocators, BoardLocators, LogoutLocators
 
+from util.data_preparation.jira_prepare_data import CLIENT, __check_rte_status
```

Comment:
Regarding Python naming conventions `__*` in the function name means that these functions are only for internal usage inside the class or module.